packages/webapp/src/components/Table/Cell/index.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import Plain from './CellTypes/Plain';
+import HoverPillOverflow from './CellTypes/HoverPillOverflow';
+import RightChevronLink from './CellTypes/RightChevronLink';
+import IconText from './CellTypes/IconText';
+import {
+  CellType,
+  KindComponentKVP,
```

Comment:
The name of this type is confusing to me tbh 😅 what does KVP stand for?

Reply:
Sorry, its gone now anyways in terms of @kathyavini Typing approach. But "Key-Value Pair" was the intent.